spark/sql/dataframe.go

```diff
@@ -1349,3 +1353,77 @@ func (df *dataFrameImpl) Summary(ctx context.Context, statistics ...string) Data
 	}
 	return NewDataFrame(df.session, rel)
 }
+
+func anyToFillNaSupportedExpressionLiteral(value any) (*proto.Expression_Literal, error) {
```

Comment:
We have the `column.NewLIteral` with the same expression.

Reply:
I wasn't sure about supporting `byte[]` there as documentation in proto doesn't mention `binary` as supported type.

Reply:
It does support `binary` and the conversion from `[]byte` to the binary type as well.

Reply:
Sorry. I meant support for binary in fillna itself. The proto and pyspark don't mention support for binary: https://spark.apache.org/docs/latest/api/python/reference/pyspark.sql/api/pyspark.sql.DataFrame.fillna.html

Reply:
`NewLiteral` returns `Expression`, while in proto we need `Literal_Expression` instead, so `NewLiteral` won't work out of the box, or we need to add checks if value is actual literal value.